connPFM/tests/test_deconvolution.py

```diff
@@ -38,3 +38,33 @@ def test_stability_lars(surr_dir):
     auc_osf = np.load(auc_filename)
     # Check if AUC is correct
     assert np.all(auc == auc_osf)
+
+
+def test_stability_lars_ME(ME_lars):
+    # Test for multiecho data
+    data = np.load(ME_lars[0])
+    first = 0
+    last = 0
+    if first is None:
+        nvoxels = 1
+    else:
+        nvoxels = last - first + 1
+        voxel = first
+
+    # Load HRF
+    hrf = np.load(ME_lars[1])
+
+    nvoxels = last - first + 1
+    nscans = int(data.shape[0] / 5)
+    auc = np.zeros((nscans, nvoxels))
+    from connPFM.deconvolution.stability_lars import StabilityLars
```

Comment:
This should go on the top of the file.

Reply:
Nope because if you put it on the top the random.seed fix will not affect the calculations inside StabilityLars functions and the tests will fail

Reply:
This was made on propouse

Reply:
In fact, I forgot to add it on the function, thanks

Reply:
But shouldn't the seed only affect `StabilityLars()` when you initialize it, rather than when you import?